packages/nimble-components/src/chip/specs/README.md

```diff
@@ -0,0 +1,200 @@
+# Nimble Chip
+
+## Overview
+
+The `nimble-chip` is a component that presents text, an optional "remove" button, optional slotted content to the left of the text, and an optional error icon.
+
+### Background
+
+- [Nimble Chip component issue](https://github.com/ni/nimble/issues/413)
+- [Chip design](https://www.figma.com/design/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?node-id=9269-229459&m=dev)
+- [Spright chat feature backlog](https://github.com/ni/nimble/issues/2610) (component for attachment chips)
+
+### Containing Library
+
+The intent is for this to be put in Nimble, for the following reasons:
+
+- This is a fairly constrained component, and thus easily ported to any needed UI platform.
+- It has broad use possibilities.
+- Designs for the component have been completed (see above link)
+
+However, there is a potential for there to be some accessibility concerns, notably around the remove button, that could persuade us to put this in Spright.
```

Comment:
I'd expect those accessibility concerns to be hashed out within this HLD rather. That would allow this to be a more concrete statement of "this will be put in nimble". I think it'll cause a lot of churn for us to move into development without knowing whether or not there are too many accessibility problems for this component to live in nimble.